clients/client-python/gravitino/exceptions/base.py

```diff
@@ -93,3 +93,11 @@ class UnauthorizedException(GravitinoRuntimeException):
 
 class BadRequestException(GravitinoRuntimeException):
     """An exception thrown when the request is invalid."""
+
+
+class NoSuchCatalogException(NotFoundException):
```

Comment:
I think it should be placed at same place as the `NotFoundException` and `AlreadyExistsException`, just for the readability, not a functional problem.

Reply:
Done!